.github/workflows/verify.yml

```diff
@@ -0,0 +1,39 @@
+name: Verify every day
+
+on:
+  schedule:
+    # every day at 9am
+    - cron: '0 9 * * *'
+
+env:
+  go-cache-name: go
+
+jobs:
+  verify:
+    name: Verify docs and spelling
+    runs-on: [ubuntu-latest]
+    steps:
+    - name: Set up Go 1.17
+      uses: actions/setup-go@v3
+      with:
+        go-version: 1.17
+    - name: Check-out code
+      uses: actions/checkout@v3
+    - name: Run verify scripts
+      run: make verify
+    - name: Checking for broken Markdown links for main branch
+      if: ${{ github.base_ref == 'main' }}
+      uses: gaurav-nelson/github-action-markdown-link-check@v1
+      with:
+        folder-path: './docs'
+        file-path: './README.md, ./CHANGELOG.md, ./CONTRIBUTING.md, ./GOVERNANCE.md, ./MAINTAINERS.md, ./ROADMAP.md, ./SECURITY.md'
+        config-file: 'hack/.md_links_config.json'
+    - name: Checking for broken Markdown links for release branch
+      if: ${{ startsWith(github.base_ref, 'release-') }}
+      uses: gaurav-nelson/github-action-markdown-link-check@v1
+      with:
+        config-file: 'hack/.md_links_config.json'
```

Comment:
I think you can remove these lines since you trigger this workflow based on a schedule instead of an event in a `release-` branch.

Reply:
Removed.